test/parallel/test-wasm-simple.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+require('../common');
+
+const assert = require('assert');
+const fixtures = require('../common/fixtures');
+
+const buffer = fixtures.readSync('test.wasm');
+
+assert.ok(WebAssembly.validate(buffer), 'Buffer should be valid WebAssembly');
+
+WebAssembly.instantiate(buffer, {}).then((results) => {
```

Comment:
Nit: `common.mustCall` around the `then` would be better.

Reply:
oh, right … we might also want `common.crashOnUnhandledRejection()` to make sure it really crashes when the promise gets rejected